tests/driver_ltc4150/main.c

```diff
@@ -96,8 +97,8 @@ static void pulse_cb(ltc4150_dev_t *dev, ltc4150_dir_t dir, uint64_t now_usec,
  */
 static void spin(uint32_t seconds)
 {
-    uint32_t till = xtimer_now_usec() + US_PER_SEC * seconds;
-    while (xtimer_now_usec() < till) { }
+    uint32_t till = ztimer_now(ZTIMER_USEC) + US_PER_SEC * seconds;
+    while (ztimer_now(ZTIMER_USEC) < till) { }
```

Comment:
overflow might occur -the old code has the same problem-; works due to short test run time < 1h:11min 

may use ztimer_spin  instead

Reply:
This makes sense to include

Reply:
done